app/models/checksum_audit_log.rb

```diff
@@ -1,20 +1,49 @@
 class ChecksumAuditLog < ActiveRecord::Base
-  # TODO: this method doesn't seem to be used. Remove?
-  def self.fixity_check_log(id, path, version_uri)
```

Comment:
In general re deprecations:  This stuff had to change so much that adding deprecations everywhere would be a major increase in complexity. I think that the former methods probably _were not used_ by anyone, because, well, they didn't really work.  So I guess I'm making the argument that the previous signatures and methods were effectively bugs. I am normally pretty pro-backwards-compat, but in this case, I don't think backwards compat with probably-unused broken architecture is a priority. 

I recognize other opinions may be reasonable though! But I don't know if I personally have time to go back and make sure all existing (public?) methods and signatures still work with deprecation.